node/options.go

```diff
@@ -111,17 +111,6 @@ func WithDataStore(store dataRetriever.StorageService) Option {
 	}
 }
 
-// WithTxSignPrivKey sets up the single sign private key option for the Node
-func WithTxSignPrivKey(sk crypto.PrivateKey) Option {
```

Comment:
You can also remove WithInitialNodesBalances from here, from tests and the n.initialNodesBalances variable from node.go

Reply:
done.